adagrams/game.py

```diff
@@ -1,11 +1,115 @@
+import random
+
+def create_letter_list():
+    #put the helper function here
+    LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+    letter_pool = []
+    for letter in LETTER_POOL:
+        for i in range(LETTER_POOL[letter]):
+            letter_pool.append(letter)
+    return letter_pool
```

Comment:
Great logic! One stylistic note: in the inner `for` loop we never use the `i` variable. To make it clear that this variable is unneeded we can use an underscore instead: `for _ in range(LETTER_POOL[letter]):`. This is a common idiom in Python when we want to show we don't care about the value of a variable.